scripts/dev/e2e.py

```diff
@@ -258,6 +245,9 @@ def parse_args() -> argparse.Namespace:
         help="Cleanup the context after executing the tests",
         action="store_true",
     )
+    parser.add_argument(
+        "--cluster-wide", help="Watch all namespaces", action="store_true",
```

Comment:
in `run.sh` we use `--clusterwide` and here `--cluster-wide`. Is this an error or am I missing something?

Reply:
That is an error! Good catch, this gave me some grief with evergreen testing. It should be fixed now.